### PR TITLE
Create contributing guidelines and license

### PR DESCRIPTION
**Pull request by firstof9**
## Proposed change

Documentation update

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR is related to issue: n/a

### CODE REVIEW

**Comment by firstof9** [comment]
@moralmunky What do you think about these guidelines and using the MIT license?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 moralmunky
```
Comment (firstof9) [inline LICENSE:3]: If these changes are acceptable I think you have to put your name here on the Copyright.

Comment (moralmunky) [reply LICENSE:3]: Looked into the difference between open source licenses. I think MIT is fine. Do you want to do something like [Chromium](https://github.com/chromium/chromium/blob/master/LICENSE)?

`// Copyright 2015 The Chromium Authors. All rights reserved.`

Then list our names and email addresses in an AUTHORS file?

Comment (firstof9) [reply LICENSE:3]: That sounds fine to me.

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# This is the list of Mail and Packages's significant contributors.
+#
+# This does not necessarily list everyone who has contributed code,
+# especially since many employees of one corporation may be contributing.
+# To see the full list of contributors, see the revision history in
+# source control.
+@moralmunky
```
Comment (firstof9) [inline AUTHORS:7]: Just need your info here @moralmunky

Comment (moralmunky) [reply AUTHORS]: Done, just waiting for the checks to complete.

**Review by moralmunky — APPROVED** [review]

**Pull request merged**